.changeset/honest-cherries-push.md

```diff
@@ -0,0 +1,5 @@
+---
+"blitz": patch
+---
+
+Fix CLI version check by only checking when generating a new blitz project & by using the global package manager
```

Comment:
Add a global Blitz version check when generating a new Blitz project to ensure users use the latest Blitz.